### DISCUSSION TITLE
Icons are not loading in clr and cds Angular project

### DISCUSSION THREAD

**Discussion by ben-str-evoila** [vmware/clarity#6371] [Q&A]
If installed the Clarity project works fine. Only the icons do not work.
The issue is that a coworker and I tried including icons, but they always turn into three dots that seem to be loading.
For some reason one or two work, f.e. info-circle.

**Comment by mathisscott** [comment]
Are you loading the cog icon into the registry [as demonstrated here](https://clarity.design/foundation/icons/#import)?

**Comment by ben-str-evoila** [comment]
Do I have to import every icon one by one? Can't I just import them all at once?

**Comment by mathisscott** [comment]
We don't recommend importing all the icons. The reason there is that it's likely a product doesn't use all of the icons. So the product's code base will be bloated with icons the product doesn't need.

If you absolutely must load many, many icons at once, [we have icon sets like in the old library](https://github.com/vmware/clarity/tree/next/packages/core/src/icon/collections). But we discourage teams from using them. 

It's a bad practice and we recommend loading the icons you need as they are needed instead.
